src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/catalog.cljs

```diff
@@ -192,7 +192,8 @@
                                           {:datumTokenizer js/Bloodhound.tokenizers.whitespace
                                            :queryTokenizer js/Bloodhound.tokenizers.whitespace
                                            :remote (clj->js {:url (str (config/api-url-root) "/duos/autocomplete/%QUERY")
-                                                             :wildcard "%QUERY"})}))]
+                                                             :wildcard "%QUERY"
+                                                             :cache false})}))]
```

Comment:
I was noticing some weird behavior here (maybe due to the new typeahead js?) so I added the cache false